core/src/test/java/org/elasticsearch/common/xcontent/ObjectParserTests.java

```diff
@@ -34,6 +35,8 @@
 
 public class ObjectParserTests extends ESTestCase {
 
+    private static ParseFieldMatcherSupplier strictParsing = () -> ParseFieldMatcher.STRICT;
```

Comment:
Will rename. The constant is only here to not having to repeat the lambda everywhere in tests in this class.